### PR TITLE
Remove never-supported HTML elements/attributes

### PR DESCRIPTION
**Pull request by saschanaz**
A checklist to help your pull request get merged faster:
- [x] Summarize your changes
- [ ] Data: link to resources that verify support information (such as browser's docs, changelogs, source control, bug trackers, and tests)
- [ ] Data: if you tested something, describe how you tested with details like browser and version
- [ ] Review the results of the linter and fix problems reported (If you need help, please ask in a comment!)
- [x] Link to related issues or pull requests, if any: #6854, #6768

### CODE REVIEW

File: html/elements/isindex.json [hunk 1]
```diff
@@ -1,148 +0,0 @@
-{
-  "html": {
-    "elements": {
-      "isindex": {
```
Comment (foolip) [inline html/elements/isindex.json:4]: Evidence for `<isindex>` removal is in https://github.com/mdn/browser-compat-data/pull/6931#discussion_r517262407.

**Review by Elchi3 — CHANGES_REQUESTED** [review]
The autocapitalize attribute seems to be in Chrome 43 and iOS Safari 5.
Source: https://developer.chrome.com/blog/autocapitalize/

Getting rid of the menu features should be fine, though.

**Comment by saschanaz** [comment]
Hmm, that's https://github.com/mdn/browser-compat-data/issues/7519. I'll exclude it for now.

**Review by Elchi3 — APPROVED** [review]
Thank you! 👍

**Pull request merged**